languages/purescript/docs/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,50 @@
-# How to implement an PureScript concept exercise
+# How to implement a PureScript concept exercise
 
-TODO: describe how to implement a concept exercise for the PureScript track. For inspiration, check out the [C# version of this file][csharp-implementing].
+This document describes the steps required to implement a concept exercise for the PureScript track. As this document is generic, the following placeholders are used:
 
-[csharp-implementing]: ../../csharp/docs/implementing-a-concept-exercise.md
+- `<SLUG>`: the name of the exercise in kebab-case (e.g. `anonymous-methods`).
+- `<NAME>`: the name of the exercise in PascalCase (e.g. `AnonymousMethods`).
+
+Before implementing the exercise, please make sure you have a good understanding of what the exercise should be teaching (and what not). This information can be found in the exercise's GitHub issue.
+
+To implement a concept exercise,  the following files need to be created:
+
+<pre>
+languages
+└── PureScript
+    └── exercises
+        └── concept
+            └── <SLUG>
+                ├── .docs
+                |   ├── instructions.md
+                |   ├── introduction.md
+                |   ├── hints.md
+                |   └── after.md (optional)
+                ├── .meta
+                |   ├── config.json
+                |   ├── bower.json
```

Comment:
I think this file should be in its parent (`<SLUG>`) directory.

Reply:
The `bower.json`?